README.md

```diff
@@ -140,9 +178,60 @@ Positive Tests:
 
 PDF report is ready: reports/waf-evaluation-report-generic-2021-March-03-15-15-51.pdf
 ```
----
 
-### Configuration options
+The report file `waf-evaluation-report-<date>.pdf` is available in the `reports` folder of the user directory.
+
+## Other options to run GoTestWAF
+
+In addition to running the GoTestWAF Docker image downloaded from Docker Hub, you can run GoTestWAF by using the following options:
+
+* Clone this repository and run the [GoTestWAF Docker image](https://hub.docker.com/r/wallarm/gotestwaf) by using the `make` utility.
+For example:
+
+    ```
+    git clone https://github.com/wallarm/gotestwaf.git
+    cd gotestwaf
+    make scan_local_from_docker
```

Comment:
This won't work if modsec is not started yet. So `make modsec` required before `make scan_local_from_docker`.

Reply:
This command will run GoTestWAF on http://127.0.0.1:8080